strawberry/flask/views.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from flask import Response, render_template_string, request
+from flask.views import View
+from graphql import graphql_sync
+from graphql.error import format_error as format_graphql_error
+from graphql.type.schema import GraphQLSchema
+
+from .playground import playground_template
+
+
+class GraphQLView(View):
+    schema = None
+
+    methods = ["GET", "POST"]
+
+    def __init__(self, schema=None):
+        self.schema = schema
+
+        if not self.schema:
+            raise ValueError("You must pass in a schema to GraphQLView")
```

Comment:
I'm wondering if we should make this non-optional if we raise an error :)

Reply:
Will do

Reply:
no, I think we can remove `=None` from the arguments :)

<img width="282" alt="image" src="https://user-images.githubusercontent.com/667029/71153171-5b137d80-2230-11ea-8927-fe5373156e53.png">

Reply:
Alright sounds good. Will do